docs/webhooks/overview.mdx

```diff
@@ -31,6 +31,8 @@ The payload of a webhook is a JSON object that contains the following properties
 - `data`: contains the actual payload sent by Clerk. The payload can be a different object depending on the `event` type. For example, for `user.*` events, the payload will always be the [User](/docs/references/javascript/user/user) object.
 - `object`: always set to `event`.
 - `type`: the type of event that triggered the webhook.
+- `timestamp`: timestamp in seconds of when the event occurred.
```

Comment:
```suggestion
- `timestamp`: timestamp in milliseconds of when the event occurred.
```

Reply:
Thanks for catching that Kevin, I blindly copy-pasted from another source.